pkg/collector/worker/worker.go

```diff
@@ -141,6 +147,13 @@ func (w *Worker) Run() {
 			continue
 		}
 
+		if !w.shouldRunIntegrationInstance(check) {
+			checkLogger.Debug("HA Integration skipped")
```

Comment:
Could you please add an explanation about why the check is skipped, in the similar fashion to how it is done on line 146?